src/components/forums/forum/ForumPageContent.tsx

```diff
@@ -57,7 +57,9 @@ export function ForumPageContent(props: ForumPageContentProps): JSX.Element {
   const onCreateForum = async (info: ForumInfo): Promise<void> => {
     setCreating(true);
     setCreationData({ title: info.title, desc: info.description });
-    const res = await forumObject.createForum(info);
+    // create new ForumObject with postboxwrapper v2
+    const forumObjectV2 = new DispatchForumV2(wallet, connection, cluster);
```

Comment:
trials the new V2 for create forum here, this will be abstracted in `DispatchProvider` later as a context

Reply:
should dispatchforumv2 get a user object instead of wall/conn/cluster?

also feels like that cluster should be wherever conn is defined.

lastly, dispatch connection should hold not only a conn to chain via the cluster etc but also to our own api server (in the future)

Reply:
> should dispatchforumv2 get a user object instead of wall/conn/cluster?
good point. not sure what the best convention here is. I was thinking just maintaining one 'DispatchForumV2 object' across the app, which creates the user object, so it's easier to pass around with context. Instead of creating a UserContext, and a DispatchForumV2Context, and having DispatchForum contain a user object as well.
> also feels like that cluster should be wherever conn is defined.
unfortunately, web3js connection doesn't infer or store which cluster the endpoint belongs to. you can see in `User.ts` the DispatchConnection object takes cluster as one of the options as well. maybe we can use DispatchConnection instead to be passed in?
> lastly, dispatch connection should hold not only a conn to chain via the cluster etc but also to our own api server (in the future)
agreed, this can be a default property of the connection.